src/maintainers.sh

```diff
@@ -160,9 +160,23 @@ function print_files_authors()
   local printed_authors_separator=false
 
   for file in "${files[@]}"; do
-    authors=$(grep --only-matching --extended-regexp 'MODULE_AUTHOR *\(.*\)' "$file" |
-      sed --regexp-extended 's/(MODULE_AUTHOR *\( *\"|\" *\))//g' |
-      sed --expression ':a' --expression 'N' --expression '$!ba' --expression 's/\n/, /g')
+    # Get every content inside the MODULE_AUTHOR's parenthesis (even if it is in multiple lines)
+    # Here grep matches 'MODULE_AUTHOR (' followed by a sequence of non-')' characters
+    # But it only matches the characters to the right of \K thanks to its look-behind nature
+    authors=$(grep --null-data --perl-regexp --only-matching 'MODULE_AUTHOR *\(\K[^)]*' "$file" |
+      tr --delete '\0')
```

Comment:
Either break the command (one line `authors=$(grep ...)` and another `authors=$(printf '%s' "$authors" | tr ...`), or make it one line.